scripts/start_webserver.sh

```diff
@@ -2,20 +2,13 @@
 
 set -e
 
-if [ ! -f "/home/indy/ledger/sandbox/pool_transactions_genesis" ] && [ -z "${GENESIS_URL}" ] && [ -z "${GENESIS_FILE}" ]; then
-  echo "Ledger does not exist - Creating genesis data..."
-	if [ ! -z "$IPS" ]; then
-		echo von_generate_transactions -s "$IPS"
-		von_generate_transactions -s "$IPS"
-	elif [ ! -z "$IP" ]; then
-		echo von_generate_transactions -i "$IP"
-		von_generate_transactions -i "$IP"
-	else
-		echo von_generate_transactions
-		von_generate_transactions
-	fi
+if [ ! -f "/home/indy/config/genesis.txn" ] && [ -z "${GENESIS_URL}" ] && [ -z "${GENESIS_FILE}" ]; then
+  bash ./scripts/init_genesis.sh
```

Comment:
The `start_webserver.sh` script, or any of the scripts, do not exist on the web server base image due to how the base images were separated.

Reply:
Fixed by reverting base images separation.